bullet/src/LinearMath/btIDebugDraw.h

```diff
@@ -102,7 +102,7 @@ class	btIDebugDraw
 	
 	virtual void	setDebugMode(int debugMode) =0;
 	
-	virtual int		getDebugMode() const = 0;
```

Comment:
Does it not build without this?

If not, does annotating the method with `[Const]` not fix things?

(Hopefully we can avoid this, as we don't like to diverge from upstream bullet - it will make upgrades harder.)

Reply:
i tried changing this back to `virtual int		getDebugMode() const = 0;` and
```
interface btIDebugDraw {
...
  [Const] long getDebugMode();
};

[JSImplementation="btIDebugDraw"]
interface DebugDrawer {
...
  [Const] long getDebugMode();
};
```
but I get
```
glue.cpp:4639:14: error: allocating an object of abstract class type 'DebugDrawer'
  return new DebugDrawer();
             ^
../src/LinearMath/btIDebugDraw.h:105:15: note: unimplemented pure virtual method 'getDebugMode' in 'DebugDrawer'
        virtual int             getDebugMode() const = 0;
```

Reply:
Assuming you're willing to pull this in with the change to btIDebugDraw.h, I believe this is good to merge. I was able to hook it up to three.js successfully (I'll release that as a separate npm package) and it looks like this: https://i.gyazo.com/1cce3aa83a231918aaae7f818f6c2e1f.png (using DBG_DrawAabb).